setup.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Read in config file and assign variables
+outputs=`python read_config.py`
+vars=$(echo $outputs | tr "(',)" "\n")
+vararray=($vars)
+
+subscription=${vararray[0]}
+res_grp_name=${vararray[1]}
+location=${vararray[2]}
+cluster_name=${vararray[3]}
+node_count=${vararray[4]}
+vm_size=${vararray[5]}
+
+# Install Azure-CLI
+curl -L https://aka.ms/InstallAzureCli
+
+# Install kubectl - Kubernetes-CLI
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+# Make the kubectl binary executable
+chmod +x ./kubectl
+# Move the binary into your PATH
+sudo mv ./kubectl /usr/local/bin/kubectl
```

Comment:
This in particular looks like a possible can of worms if there hasn't been a check earlier on to see if there are other kubectl versions available :-)